### PR TITLE
added models package and Card, Suit and Game classes

### PR DESCRIPTION
**Pull request by charan96**

### CODE REVIEW

**Comment by dennislee15** [comment]
add more content to the files bruh

**Comment by charan96** [comment]
nah...u don't get it bruh....i'm just giving structure to the project bruh.... now we can create issues to work on this stuff man

**Comment by dennislee15** [comment]
at least bring in the basic functionality from the previous project so we don't waste time writing the same stuff

File: src/main/java/models/Card.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package models;
+
+/**
+ * Created by ramcharan on 3/9/16.
+ */
+public class Card
+{
```
Comment (dennislee15) [inline src/main/java/models/Card.java:10]: basic functionality, such as public final int value, getValue(), getSuit()

**Comment by charan96** [comment]
okay...i'll be back

**Pull request closed without merging**